### PR TITLE
Use curl instead of wget

### PR DESCRIPTION
**Pull request by bkmgit**
Curl is installed on GitBash but wget is not by default, so it is helpful to use curl as it will make the lesson easier to use from a variety of shell environments.

### CODE REVIEW

**Comment by tkphd** [comment]
`wget` is installed by default in Linux environments, but `curl` is not. We'll need a better solution than only using one or the other.

File: _episodes/15-transferring-files.md [hunk 1]
```diff
@@ -19,16 +19,17 @@ cover here.
 
 ## Download files from the Internet
 
-One of the most straightforward ways to download files is to use `wget`. Any
+One of the most straightforward ways to download files is to use `curl`.  Any
```
Comment (mikerenfro) [inline _episodes/15-transferring-files.md:22]: Remove extra spaces around `Any`?

**Review by mikerenfro — CHANGES_REQUESTED** [review]
No arguments in principle, since both Git Bash and RHEL-derived OSes install `curl` by default, but not `wget`. Not sure how prevalent other non-RHEL OSes are in HPC, or which ones include `wget` but not `curl` by default.

Maybe there's room for a callout or other note where we tell the learner that the equivalent to `curl -O` is `wget`? They could reasonably Google it or use man pages, but might be worth adding it explicitly.

None of the above would block merging, so I'm only marking this as "Request changes" for the one set of extra spaces on line 22. If I can edit that myself, I do so, then mark this as "Approved", and wait a day or so for others' review.

**Review by mikerenfro — APPROVED** [review]

**Comment by mikerenfro** [comment]
> 
> 
> `wget` is installed by default in Linux environments, but `curl` is not. We'll need a better solution than only using one or the other.

Did not see this before I started my review. Will verify this one way or the other, since that's not my recalled experience.

**Comment by tkphd** [comment]
Amended my comment: CTCMS uses Debian extensively, where `curl` is not installed by default. 
I'm not opposed to switching up the lesson, but we would have to make a note somewhere -- perhaps in a Workshop Preparations checklist -- to ensure `curl` is installed beforehand, or create a callout for "`curl` not working? Try `wget`!"

**Comment by tkphd** [comment]
Hmm, since these commands are running on the local device (laptop, etc), we have no control over package installation. 
We should add a note in Setup for users to check `which curl` and install if absent.

**Comment by mikerenfro** [comment]
> Hmm, since these commands are running on the local device (laptop, etc), we have no control over package installation.
> We should add a note in Setup for users to check `which curl` and install if absent.

It's mostly on the local laptop, but there's one instance on the HPC, where we're checking if they can download files directly, or if the system has been firewalled off from direct downloads.

**Comment by tkphd** [comment]
Right. We need workshop (#347) and local(#348) configuration guidance.

**Comment by bkmgit** [comment]
Modifying to include both curl and wget

**Comment by tkphd** [comment]
Thank you, @bkmgit -- I appreciate all the work you're putting into the lesson!

**Pull request merged**